java-generic-mqtt/src/main/java/com/cumulocity/generic/mqtt/client/converter/AvroMessageConverter.java

```diff
@@ -0,0 +1,39 @@
+package com.cumulocity.generic.mqtt.client.converter;
+
+import org.apache.pulsar.client.api.Schema;
+import org.apache.pulsar.client.api.SchemaSerializationException;
+
+/**
```

Comment:
question: is this specifically an Avro converter? If so, we should say so.

Reply:
Done.